gapic/api/artman_monitoring.yaml

```diff
@@ -15,5 +15,15 @@ java:
   final_repo_dir: ${REPOROOT}/google-cloud-java/google-cloud-monitoring
 php:
   final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-monitoring
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-monitoring
+go:
+  final_repo_dir: ${REPOROOT}/gapi-monitoring-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-monitoring
 ruby:
   final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-monitoring
+  skip_packman: True
+nodejs:
+  final_repo_dir: ${REPOROOT}/gcloud-node/packages/monitoring
```

Comment:
This will be 'google-cloud-node' since the package name has been changed (I've created another PR of updating other configs, see https://github.com/googleapis/googleapis/pull/128)